config/default.json

```diff
@@ -62,5 +62,5 @@
     "0xd03ea8624c8c5987235048901fb614fdca89b117"
   ],
   "dappMailerUrl": "http://127.0.0.1:3050",
-  "dappMailerSecret": ""
+  "dappMailerSecret": "xxMv8a13I3YOZSKq9XmX285N0o4m1qHyKjgQWb2g83daGtPI4VmEw7dImu8kvO2ovG4EMC1bvjO906o365BaJBZtArpnxJCoYsCJ"
```

Comment:
did you mean to commit this?

Reply:
Not really, but it doesn't matter and kind of convenient. It's a local thing anyway